### PR TITLE
SC: Add API for get balance of operators

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes
This PR introduce APIs for getting balance of operators.

```
> subbridge.parentOperatorBalance
1e+50
> subbridge.childOperatorBalance
1e+50
```

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by KimKyungup** [comment]
@jeonghonah @andybclee @nogang PTAL. cc @laggu

File: node/sc/local_backend.go [hunk 1]
```diff
@@ -70,6 +70,14 @@ func (lb *LocalBackend) CodeAt(ctx context.Context, contract common.Address, blo
 	return statedb.GetCode(contract), nil
 }
 
+func (lb *LocalBackend) BalanceAt(ctx context.Context, account common.Address, blockNumber *big.Int) (*big.Int, error) {
+	if blockNumber != nil && blockNumber.Cmp(lb.subbrige.blockchain.CurrentBlock().Number()) != 0 {
```
Comment (nogang) [inline node/sc/local_backend.go]: If blockNumber is nil, don't you handle it?

Comment (nogang) [reply node/sc/local_backend.go]:
```suggestion
	if blockNumber == nil || blockNumber.Cmp(lb.subbrige.blockchain.CurrentBlock().Number()) != 0 {
```

Comment (KimKyungup) [reply node/sc/local_backend.go]: If the blockNumber is nil, this will use current block. :) Thanks.

**Pull request merged**